### ISSUE TITLE
12.5 (10.a) imprecise wording, "floating point" is not an ancestor of a type

### ISSUE THREAD

**Issue by beemirang** [pat-rogers/Ada-202x-WG9-Informal-Review#39]
"Ramification: The term “formal floating point type” refers to a type defined by a formal_floating_point_definition. It does not include a formal derived type whose ancestor is {a} floating point {type}. "

**Comment by ARG-Editor** [comment]
Agreed. Added to the AARM AI (AI12-0005-1) and fixed in the working draft.
